services/orders_item_remove/lambda_function.py

```diff
@@ -28,8 +28,11 @@ def lambda_handler(event, context):
 
         response = orders_table.update_item(
             Key={'id': order_id},
-            UpdateExpression="set #tms = :items",
-            ExpressionAttributeValues={':items': order_items},
+            UpdateExpression="SET #tms = :items ADD total_cost :cost",
```

Comment:
it's advised to not use ADD, it's better to use SET for these types of operations